Device: cuda
CUDA Version: 12.1
cuDNN Version: 8902
PyTorch Version: 2.3.0

Model used: PACSCNN_4
Initial dataset creation
Initial dataset creation
Initial loss: 0.6921877264976501
<__main__.Policy object at 0x14794e030410>
0.6921877264976501
Round 1 took 0.40033888816833496s, results:
{'t': 0, 'current_accuracy': 0.75, 'current_loss': 0.6601464450359344, 'train_loss': None, 'decision': 0, 'drift_rate': 0.0, 'target_domains': ['photo']}
Traceback (most recent call last):
  File "/home/apiasecz/programming/concept_drift_optimal_adaptation_FL/src/execute/evaluate_policy.py", line 899, in <module>
    main()
  File "/home/apiasecz/programming/concept_drift_optimal_adaptation_FL/src/execute/evaluate_policy.py", line 879, in main
    evaluate_policy_under_drift(
  File "/home/apiasecz/programming/concept_drift_optimal_adaptation_FL/src/execute/evaluate_policy.py", line 663, in evaluate_policy_under_drift
    drift_rate, target_domains = drift_scheduler.get_drift_params(t)
                                 ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/apiasecz/programming/concept_drift_optimal_adaptation_FL/src/execute/evaluate_policy.py", line 452, in get_drift_params
    self.limits_interval = (self.burst_interval[0], self.burst_interval[1])
                            ~~~~~~~~~~~~~~~~~~~^^^
TypeError: 'int' object is not subscriptable
